components/studio/bin/hab-studio.ps1

```diff
@@ -301,12 +301,22 @@ function Enter-Studio {
         Start-Process "$env:STUDIO_SCRIPT_ROOT\powershell\pwsh.exe" -ArgumentList "-Command `"& {Get-Content $env:HAB_STUDIO_ENTER_ROOT\hab\sup\default\out.log -Tail 100 -Wait}`""
       }
       else {
-        Get-Content $env:HAB_STUDIO_ENTER_ROOT\hab\sup\default\out.log -Tail 100 -Wait
+        # Container studios run habitat as a service which logs to a 
+        # configured file location
+        $svcPath = Join-Path $env:SystemDrive "hab\svc\windows-service"
+        [xml]$configXml = Get-Content (Join-Path $svcPath HabService.exe.config)
```

Comment:
This is making me wonder what a `hab svc path <IDENT>` subcommand might look like, just for scripting convenience

Reply:
yeah fur sure